src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionActions.tag

```diff
@@ -74,37 +74,15 @@
     data-placement="top"
     style="display: inline-block; padding-right: 5px;">
   <div class="btn-group margin-bottom-7px">
-    <a class="btn btn-default btn-xs btn-tm-actions session-remind-for-test<c:if test="${not actions.allowedToRemind || actions.privateSession}"> disabled</c:if>"
-        href="${actions.allowedToRemind && not actions.privateSession ? actions.remindLink : 'javascript:;'}"
-        data-fsname="${actions.fsName}">
-      Remind
-    </a>
     <button type="button"
-        class="btn btn-default btn-xs btn-tm-actions dropdown-toggle session-remind-options-for-test"
-        data-toggle="dropdown"
-        aria-expanded="false"
+        class="btn btn-default btn-xs btn-tm-actions session-remind-particular-for-test"
+        data-actionlink="${actions.remindParticularStudentsPageLink}"
+        data-courseid="${actions.courseId}"
+        data-fsname="${actions.fsName}"
+        data-toggle="modal"
+        data-target="#remindModal"
         <c:if test="${not actions.allowedToRemind || actions.privateSession}">disabled</c:if>>
-      <span class="caret"></span>
+      Remind
     </button>
-    <ul class="dropdown-menu" role="menu">
-      <li>
-        <a href="${actions.remindLink}"
```

Comment:
Is this variable still relevant? If not, can get rid of it.

Reply:
Done!